doc/METADATA.ini

```diff
@@ -1060,6 +1060,14 @@ description= who should see this configuration setting?
 	- developer: settings only relevant for developers of the application
 	- internal: settings which should not be seen by anyone. (e.g. intermediate steps for calculations)
 
+[check/rgbcolor]
+type= enum
+	any
```

Comment:
AFAIK this means that the only valid value for `check/rgbcolor` is `any`. Use `type= any` instead.

Reply:
Or type=string. Or do we really want to support binary data?

Reply:
The code below only checks that the metakey exists, so the value actually doesn't matter.

Reply:
Then it should be "empty". (like [binary], see #2705)

Reply:
The enum was still leftover from an earlier version. As @kodebach pointed out, the keys value doesn't matter, only its existence. I set the type to empty.